l10n_es_aeat_verifactu/models/account_move.py

```diff
@@ -378,5 +379,19 @@ def _get_receiver_dict(self):
             }
         }
 
+    @api.model
+    def _get_verifactu_qr_values(self):
+        """Get the QR values for the verifactu"""
+        self.ensure_one()
```

Comment:
aquí, si es api.model , habría que quitar el self.ensure.one(),  o quitar el api.model y dejar el self.ensure_one()
```suggestion
    @api.model
    def _get_verifactu_qr_values(self):
        """Get the QR values for the verifactu"""
```

Reply:
Es cierto. No debería ser @api.model, ya que depende de los valores de un registro específico para ofrecer resultados.

Reply:
De la misma manera, no sería adecuado usar @api.model, por ejemplo, en _get_verifactu_hash_string, ya que este método también depende de valores específicos del modelo.

Reply:
tienes razón, hemos actualizado nuestra rama y modificado eso. Tendrás que actualizarte tu rama para que no te salgan conflictos. Con eso ya estaría todo ok para mergear  :)  Buen trabajo